Business Social Network/AppModules/Search/views/searchBox.html

```diff
@@ -16,35 +14,53 @@
 
 <div ng-controller = "SearchBoxController">
 
+  <div class="col-lg-6">  <!-- whole search box and both buttons -->
 
-  <!-- <br /><br /> -->
-
-  <div class="col-lg-6">  <!-- whole search box and button -->
+    <!-- Category button -->
 
     <div class="input-group">
 
-      <!-- input box -->
-      <input type="text" ng-model = "search_term" ng-bind = "search_term" id = "search_box"
-      class="form-control" ng-change = "update()" placeholder="What are you looking for?">
+      <div class="input-group-btn">
 
-      <!-- search button-->
-      <span class="input-group-btn">
+        <button type="button" class="btn btn-default dropdown-toggle"
+        data-toggle="dropdown" aria-haspopup="true"
+        aria-expanded="false">
 
-        <button type="button" class="btn btn-default" ng-click = "search()" >
-          <span class="glyphicon glyphicon-search" aria-hidden="true"></span> Search!
-        </button>
+        {{selected}}
 
-      </span>
+        <span class="caret"></span></button>
 
-    </div>
+        <ul class="dropdown-menu">
 
+          <li>
+            <a ng-click = "updateCategory('All Categories')">
+              All Categories</a>
+            </li>
 
-  </div>
-  <br /><br />
+            <li role="separator" class="divider"></li>
+
+            <li ng-repeat = "category in categories">
+              <a  ng-click = "updateCategory(category)">{{category}}</a>
```

Comment:
@Swailem95 
please user ng-bind instead of {{}} as this will not always work 
http://stackoverflow.com/a/16126174